src/page-modules/assistant/line-filter/index.tsx

```diff
@@ -2,44 +2,69 @@ import LabeledInput, { LabeledInputProps } from '@atb/components/labled-input';
 import { Typo } from '@atb/components/typography';
 import { getOrgData } from '@atb/modules/org-data';
 import { PageText, useTranslation } from '@atb/translations';
-import { ChangeEvent, useState } from 'react';
+import { ChangeEvent, useEffect, useState } from 'react';
 import style from './line-filter.module.css';
+import useSWR from 'swr';
 type LineFilterProps = {
   filterState: string[] | null;
   onChange: (lineFilter: string[] | null) => void;
 };
 
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
```

Comment:
Can be removed, I think.

Reply:
Mhm, the fetch operation does not work when I only remove the fetcher. Am I supposed to replace it with something?

Reply:
See `useMapTariffZones` or `useNonTransitTrip` for examples.